testing/tests/index.test.js

```diff
@@ -3,8 +3,14 @@ const path = require("path");
 
 const cheerio = require("cheerio");
 
+const { cli } = require("../../content");
+
 const buildRoot = path.join("..", "client", "build");
```

Comment:
It'd be nice to get rid of this and just use the default. That would mean the built pages would stay within the `testing` workspace as `testing/client/build/...`, and we'd have to point `yarn start:functional` to that location. But I think that's nice to keep everything related to testing within testing.